requests/__init__.py

```diff
@@ -43,6 +43,7 @@
 import urllib3
 import chardet
 import warnings
+import sys
```

Comment:
Sorry before we squash this import is also not being used anymore.

Reply:
I *think* I figured out how to squash, but I might be wrong. If there are some weird commits, that's probably why.